src/briefcase/integrations/android_sdk.py

```diff
@@ -999,6 +999,27 @@ def start_emulator(self, avd):
             start_new_session=True,
         )
 
+        # wrap AVD name in quotes since '@' is a special char on Windows
+        emulator_command = " ".join(
+            f'"{arg}"' if arg.startswith("@") else arg
```

Comment:
Is it special in cmd, or only in PowerShell? If it's only PowerShell, then the comment should say that.

Reply:
It is reserved in PowerShell; cmd treats it as a normal character. I'll update the comment.

FWIW, while confirming this, I realized the `-avd` option can be used in lieu of `@`.